nav2_graceful_controller/include/nav2_graceful_controller/parameter_handler.hpp

```diff
@@ -79,6 +79,11 @@ class ParameterHandler
 
   Parameters * getParams() {return &params_;}
 
+  /**
+  * @brief Registers callbacks for dynamic parameter handling.
+  */
+  void activateDynamicParametersCallback();
```

Comment:
You could also just make this `activate()` and make a complimentary `deactivate` for the `on_deactivate` callback so this looks like the other members